src/opm/parser/eclipse/share/keywords/000_Eclipse100/A/AQUCT

```diff
@@ -2,7 +2,7 @@
  "items" : [
  {"name" : "AQUIFER_ID" , "value_type" : "INT"},
  {"name" : "DAT_DEPTH"  , "value_type" : "DOUBLE", "dimension" : "Length"},
- {"name" : "P_INI"      , "value_type" : "DOUBLE", "dimension" : "Pressure", "default" : 0.0},
+ {"name" : "P_INI"      , "value_type" : "DOUBLE", "dimension" : "Pressure", "default" : -1.0},
```

Comment:
The `P_INI` item does not have a default value; so instead of changing it from `0.0` to `-1.0` the default value should be removed.

Reply:
P_INI also has a default value, which can only be calculated from equilibrium conditions, which can only be done, if I go through the calculations at the opm-simulator.